interface-definitions/service_dns_forwarding.xml.in

```diff
@@ -793,6 +793,184 @@
+                    <properties>
+                      <help>Zone source</help>
+                    </properties>
+                    <children>
+                      <leafNode name="axfr">
+                        <properties>
+                          <help>DNS server address</help>
+                          <valueHelp>
+                            <format>ipv4</format>
+                            <description>IPv4 address</description>
+                          </valueHelp>
+                          <valueHelp>
+                            <format>ipv6</format>
+                            <description>IPv6 address</description>
+                          </valueHelp>
+                          <constraint>
+                            <validator name="ip-address"/>
+                          </constraint>
+                        </properties>
+                      </leafNode>
+                      <leafNode name="url">
+                        <properties>
+                          <help>Source URL</help>
+                          <valueHelp>
+                            <format>url</format>
+                            <description>Zone file URL</description>
+                          </valueHelp>
+                          <constraint>
+                            <validator name="url" argument="--scheme http --scheme https"/>
+                          </constraint>
+                        </properties>
+                      </leafNode>
+                    </children>
+                  </node>
+                  <node name="options">
+                    <properties>
+                      <help>Zone caching options</help>
+                    </properties>
+                    <children>
+                      <leafNode name="timeout">
+                        <properties>
+                          <help>Zone retrieval timeout</help>
+                          <valueHelp>
+                            <format>u32:1-3600</format>
+                            <description>Request timeout in seconds</description>
+                          </valueHelp>
+                          <constraint>
+                            <validator name="numeric" argument="--range 1-3600"/>
+                          </constraint>
+                        </properties>
+                        <defaultValue>20</defaultValue>
+                      </leafNode>
+                      <node name="refresh">
+                        <properties>
+                          <help>Zone caching options</help>
+                        </properties>
+                        <children>
+                          <leafNode name="on-reload">
+                            <properties>
+                              <help>Retrieval zone only at startup and on reload</help>
+                              <valueless/>
+                            </properties>
+                          </leafNode>
+                          <leafNode name="interval">
```

Comment:
From a CLI perspective (both completion help and constraint) I think it would be easier to just make this seconds only

Reply:
the first version used seconds, but was reworked after this comment: 
https://github.com/vyos/vyos-1x/pull/3896#discussion_r1701778118

@dmbaturin what do you think about this?

Reply:
Let's just do seconds for ease of validation. We can later add suffixed values, once we have a validator for them. It shouldn't be a problem because a number without any suffix is interpreted as seconds anyway, so we'll only need to relax the validation to allow human-readable values like `1d12h`.